ihmc-java-toolkit/src/main/java/us/ihmc/tools/time/FrequencyCalculator.java

```diff
@@ -17,8 +14,9 @@
  */
 public class FrequencyCalculator
 {
-   private double frequency;
-   private final Deque<Long> pingTimes = new LinkedList<>();
+   private double alpha = 0.3;
```

Comment:
If you like this, I'll make the alpha an optional parameter and remove the decay API.